lib/IRGen/GenDecl.cpp

```diff
@@ -3350,6 +3355,47 @@ llvm::Constant *IRGenModule::emitSwiftProtocols() {
   if (SwiftProtocols.empty())
     return nullptr;
 
+  StringRef sectionName;
+  switch (TargetInfo.OutputObjectFormat) {
+  case llvm::Triple::UnknownObjectFormat:
+    llvm_unreachable("Don't know how to emit protocols for "
+                     "the selected object format.");
+  case llvm::Triple::MachO:
+    sectionName = IRGen.Opts.EmitDeadStrippableSymbols
+        ? "__TEXT, __swift5_protos, regular, live_support"
+        : "__TEXT, __swift5_protos, regular, no_dead_strip";
+    break;
+  case llvm::Triple::ELF:
+  case llvm::Triple::Wasm:
+    sectionName = "swift5_protocols";
+    break;
+  case llvm::Triple::XCOFF:
+  case llvm::Triple::COFF:
+    sectionName = ".sw5prt$B";
```

Comment:
One question I have is if this information around sectionName is something that is also somewhere else. Seems like we should have that centralized so it isn't out of sync, no?

Reply:
I guess it was just below? Nm.